federation/registry/cluster/strategy_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/kubernetes/federation/apis/federation"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
I moved the `coreFuncs` to `pkg/api/fuzzer`, and thus `pkg/api/testing` no longer imports `pkg/api/testapi`(which registers all the kubernetes groups).

and I think it is better to make it explicit that these `strategy_test.go` requires `pkg/api/testapi`, so I add it here with an underscore import.